GANDLF/cli/data_split_saver.py

```diff
@@ -0,0 +1,50 @@
+from typing import Union
+import os
+
+import pandas as pd
+from GANDLF.utils import get_dataframe, split_data
+
+
+def split_data_and_save_csvs(
+    input_data: Union[pd.DataFrame, str], output_dir: str, parameters: dict
+) -> None:
+    """
+    Split the data into training, validation, and testing sets and save them as csvs in the output directory
+
+    Args:
+        input_data (Union[pd.Dataframe, str]): The input data to be split and saved.
+        output_dir (str): The output directory to save the split data.
+        parameters (dict): The parameters dictionary.
+    """
+
+    full_data = get_dataframe(input_data)
+
+    dataframe_split = split_data(full_data, parameters)
+
+    for (
+        testing_and_valid_indeces,
+        trainingData,
+        validationData,
+        testingData,
+    ) in dataframe_split:
+
+        # training and validation dataframes use the same index, since they are based on the validation split
+        training_data_path = os.path.join(
```

Comment:
```suggestion
    ) in dataframe_split:
        # training and validation dataframes use the same index, since they are based on the validation split
        training_data_path = os.path.join(
```

Reply:
Thanks! I wonder why this was not getting picked up locally...